headertest/verify_test.go

```diff
@@ -95,15 +95,6 @@ func TestVerify(t *testing.T) {
 			},
 			err: header.ErrKnownHeader,
 		},
-		{
-			trusted: trusted,
-			prepare: func() *DummyHeader {
-				untrusted := next()
-				untrusted.HeightI += 100000
-				return untrusted
-			},
-			err: header.ErrHeightFromFuture,
```

Comment:
We should also remove this error as its unused now

Reply:
Nice catch! Fixed.